pulsar/consumer_partition.go

```diff
@@ -143,10 +144,61 @@ type partitionConsumer struct {
 	dlq         *dlqRouter
 
 	log log.Logger
-
+  providersMutex       sync.RWMutex
```

Comment:
whitespace

Reply:
Run `go fmt` to fix all formatting issues